sheepdog/globals.py

```diff
@@ -224,6 +224,5 @@ def submitted_state():
 
 # Below is the list of node states that are treated as "released"
 RELEASED_NODE_STATES = [
-    'released',
-    'live',
+    'released'
```

Comment:
Probably need to make sure removing live state won't impact current system

Reply:
Yea, checked the data we have currently, all nodes in state `live` are `File` or `Archive` and we sure don;t want to version those